frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -26,7 +26,7 @@ export const UnsubscribeSurveyModal = ({
         (product.type == 'product_analytics' &&
             (product as BillingProductV2Type)?.addons?.filter((addon) => addon.type === 'data_pipelines')[0]
                 ?.subscribed) ||
-        billing?.subscription_level === 'paid'
+        (billing?.subscription_level === 'paid' && !isAddonProduct)
```

Comment:
I feel like this is unnecessarily complicated somehow. Now with sub-all-products we need to show this if:
- The are unsubbing from pipelines directly
- OR they are unsubbing completely

I think the above does that check but in a convoluted way. Maybe the convolution is because we technically still support sub-individual-products?

Reply:
We need to remember not everyone is on ` sub-all-products`, there will be a few customers on custom plans so these checks are still needed even though most customers will be on it. 

`billing?.subscription_level === 'paid'` is basically in place of all products, the product.type passed in is actually `platform_and_support` since it requires a product.